src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
```

Comment:
There is a typo in the class name 'navigation__wraper'. It should be 'navigation__wrapper'.